lib/money_column/active_record_hooks.rb

```diff
@@ -5,11 +5,13 @@ def self.included(base)
     end
 
     module ClassMethods
```

Comment:
That would require every table missing a currency column to add one. Let's add this deprecation notice in a future PR as to not crowd the logs too much